services/request.service.ts

```diff
@@ -1,7 +1,7 @@
 // originally coppied from pages
 
-//This is here just for testing. It should eventually be put in a separate git repo
-//or be bundled with a base ASWWU project.
+// This is here just for testing. It should eventually be put in a separate git repo
```

Comment:
I think we can just get rid of these and any other comments that aren't useful to understanding the code (as well as commented out code)